pysumma/simulation.py

```diff
@@ -14,25 +17,27 @@ class Simulation(object):
 
     manager: FileManager = None
     decisions: Decisions = None
-
-    def __init__(self, executable, filemanager,
-                 run_suffix='_pysumma_run', summa_code=None):
-        """
-        Initialize a new simulation object
-        """
+    output_control: OutputControl = None
+    local_param_info: LocalParamInfo = None
+    basin_param_info: LocalParamInfo = None
+    force_file_list: ForceFileList = None
+    local_attributes: xr.Dataset = None
+    parameter_trial: xr.Dataset = None
+
```

Comment:
The changes in class, module, and function naming is to keep things in line with python coding conventions. This makes it easier for new contributors to understand what is going on, since the style is the same as almost all other python projects.